catroid/src/main/res/menu/menu_script_activity.xml

```diff
@@ -81,6 +81,7 @@
     <item
         android:id="@+id/comment_in_out"
         android:showAsAction="never"
-        android:title="@string/comment_in_out"/>
+        android:title="@string/comment_in_out"
+        android:visible="false"/>
```

Comment:
@sjaindl , it's very kind of you to fix a problem that I introduced. :smile: 
However, I already fixed that problem in PR #1915 which is why you can remove these lines again from this PR.
Thanks to @bernadettespieler for pointing this out.